.circleci/config.yml

```diff
@@ -134,24 +134,31 @@ workflows:
   test-suite:
     jobs:
       - initial-setup
+         filters:
+	   branches:
+	     ignore:
+	       - sicer-integration
```

Comment:
I think the tabs here are causing circleci to get confused

Reply:
yup, it's just a goofy, initially-accidental way to get circle to abandon the test.